pyomo/gdp/plugins/chull.py

```diff
@@ -470,22 +468,21 @@ def _transformDisjunctionData(self, obj, transBlock, index):
             disaggregationConstraint.add(
                 consIdx,
                 var == disaggregatedExpr)
```

Comment:
I think this should be changed.  Now that the new Sets are in, you can declare the disaggregationConstraint as:
```python
Constraint(NonNegativeIntegers, Any)
```
Then, this while mess (including the computation of the `consIdx` can be simplified to:
```python
disaggregationConstraint.add((i, index), var==disaggregatedExpr)
```